src/Audit.js

```diff
@@ -23,6 +23,20 @@ module.exports = class Audit {
     }
   }
 
+  hasVulnerabilities(scanResult) {
+    const vulnerableAudit = scanResult.lhr.audits['no-vulnerable-libraries']
+
+    if (
+      vulnerableAudit.details &&
+      vulnerableAudit.details.items &&
+      vulnerableAudit.details.items.length > 0
```

Comment:
@lirantal You might want to use [optional chaining Babel plugin](https://babeljs.io/docs/en/next/babel-plugin-proposal-optional-chaining.html) to refactor this at a later point.

Reply:
I like this idea, but I'm not planning to implement this at the moment.  It reminds me of using optional chaining in Swift to avoid the [pyramid of doom](https://en.wikipedia.org/wiki/Pyramid_of_doom_(programming)).

I'm a babel newb, so I'm also not sure what that plugin being classified as 'experimental' means for the long term.

Reply:
Agree. I don't want to add further complexity here and this one-off is alright with me at this point.

Reply:
That's why I wrote „might want” and „at a later point”.

@mattorb Nothing against [Hadouken indentation](https://www.elastic.co/guide/en/elasticsearch/client/net-api/current/hadouken-indentation.jpg), mkay? :-D